chipsec/helper/rwe/__init__.py

```diff
@@ -20,7 +20,7 @@
 #
 
 import platform, os
-if "windows" == platform.system().lower() and os.environ.has_key("USERWE"):
+if "windows" == platform.system().lower() and (os.path.isfile(os.path.join("C:\\Windows\\System32\\drivers\\RwDrv.sys")) or os.path.isfile(os.path.join(os.getcwd(), "win7_amd64\RwDrv.sys"))):

```

Comment:
driver should be in the system32/drivers directory if rwe is installed through the normal installer... at least, that's what happened in my system and why I added it in the first place. 

Agree on the join for the path of the chipsec location. I'll commit a fix soon.